source/common/router/config_utility.cc

```diff
@@ -0,0 +1,40 @@
+#include "config_utility.h"
```

Comment:
OK not a big deal but in the future I would do a separate code move PR, just easier to review.